Jenkinsfile

```diff
@@ -0,0 +1,265 @@
+def private_key = 'dev-key'
+def vault_pass = 'vault-pass'
+def aws_creds = 'cbj-deploy'
+
+pipeline {
+  agent {
+    node {
+      label ''
+      customWorkspace 'blue-button-deploy'
+    }
+  }
+
+  parameters {
+    string(
+      defaultValue: "",
+      description: 'The branch of the application repo to deploy. Required.',
+      name: 'BRANCH'
+    )
+    string(
+      defaultValue: "*/master",
+      description: 'The branch of the deployment repo to use for deployment.',
+      name: 'DEPLOY_BRANCH'
+    )
+    choice(
+      choices: 'no\nyes',
```

Comment:
is there a way to default this to `no`?
I've always thought of migrations as risky things, any idea what's currently set up to revert to a backup if one goes wonky?

Reply:
Good questions. This will default to `no` since it is the first choice listed. I'm not sure what the recovery process looks like if something goes wrong.